packages/sdk-js/src/auto-wrapper.ts

```diff
@@ -1,12 +1,29 @@
-import { Context, GrowthBook } from "./index";
-
+import Cookies from "js-cookie";
+import {
+  BrowserCookieStickyBucketService,
+  Context,
+  FeatureApiResponse,
+  GrowthBook,
+  LocalStorageStickyBucketService,
+  StickyBucketService,
+} from "./index";
+
+type WindowContext = Context & {
+  uuidCookieName?: string;
+  uuidKey?: string;
+  uuid?: string;
+  persistUuidOnLoad?: boolean;
+  useStickyBucketService?: "cookie" | "localStorage";
+  stickyBucketPrefix?: string;
```

Comment:
We should support most of these on the data context as well.  Outside of edge, I think being able to add `sticky-buckets="localstorage"` to the script tag will be really useful to people.